### ISSUE TITLE
Email server update

### ISSUE THREAD

**Issue by s39n** [fiberpunk1/OctoPrint-Nexus-AI#4]
i just made a new email account and i can see the Microsoft is not using a new smtp server
Server name: smtp.office365.com
Port: 587
Encryption method: STARTTLS

**Comment by fiber-punk** [comment]
I tested and smtp-mail.outlook.com works in some areas. But now many have been replaced by smtp.office365.com, smtp.office365.com will be faster.

I've added support for smtp.office365.com, just need to re-download NexusAI.
![image](https://user-images.githubusercontent.com/85388178/196437627-e760a725-dab2-46bd-8794-52a9b6529825.png)

**Comment by s39n** [comment]
I dont know why but I'm getting this error

  ![2022-10-18 12_39_09-Window](https://user-images.githubusercontent.com/22017525/196528680-e11aba95-798a-47fd-9427-3d20fa811088.png)
  
![image](https://user-images.githubusercontent.com/22017525/196529507-5341d4d3-47c2-42ba-a61e-c8d71ab9ab1b.png)

**Comment by troyhowarth** [comment]
This may not be the same issue - but yesterday when I was setting it up I received the same error message. I clicked "save settings" exited the program and reopened the program, went back into the email area and clicked "send test" and it worked even though I hadn't changed my email address/password. Thought it might have been a way it was calling the variables.

**Comment by s39n** [comment]
> This may not be the same issue - but yesterday when I was setting it up I received the same error message. I clicked "save settings" exited the program and reopened the program, went back into the email area and clicked "send test" and it worked even though I hadn't changed my email address/password. Thought it might have been a way it was calling the variables.

Are you using "smtp.office365.com"

**Comment by troyhowarth** [comment]
> Are you using "smtp.office365.com"

nah, i'm using smtp-mail.outlook.com. When i get home from work I'll download the latest release and try smtp.office365.com. I'm in Australia if geolocation is an issue with what i have/you have access to from microsoft's servers.

**Comment by s39n** [comment]
> > Are you using "smtp.office365.com"
> 
> nah, i'm using smtp-mail.outlook.com. When i get home from work I'll download the latest release and try smtp.office365.com. I'm in Australia if geolocation is an issue with what i have/you have access to from microsoft's servers.

did you get to try?

**Comment by troyhowarth** [comment]
> > > Are you using "smtp.office365.com"
> > 
> > 
> > nah, i'm using smtp-mail.outlook.com. When i get home from work I'll download the latest release and try smtp.office365.com. I'm in Australia if geolocation is an issue with what i have/you have access to from microsoft's servers.
> 
> did you get to try?

Hey,
Sorry in the delay in responding. I did try with the latest version and it's not working. Not even the older " smtp-mail.outlook.com " from the drop down. I went back to the version dated 17th of October and that one worked using " smtp-mail.outlook.com". 

In saying that. It's only partially working. It sends a test email but it isn't sending an email when it has detected a print fail.

**Comment by troyhowarth** [comment]
> I did try with the latest version and it's not working. Not even the older " smtp-mail.outlook.com " from the drop down. I went back to the version dated 17th of October and that one worked using " smtp-mail.outlook.com".

Interestingly I tried copying the config.ini file generated from the server dated 17th across to the latest release and I can now send test emails. I think potentially the new version isn't saving to the config file correctly? The server still isn't generating emails when it detects a fail though.

**Comment by fiber-punk** [comment]
herre is the Nexus AI Server's code: https://github.com/fiberpunk1/NexusAIServer/tree/main/src/email
I believe the email sending function is often related to the network configuration and email server configuration.

**Comment by ChrisMayhew** [comment]
I had this issue but I solved it by clicking save first and then trying the test email.
